Dockerfile.test-tools

```
@@ -0,0 +1,24 @@
+# based on build-tools but adds postgresql-client and postgresql packages
+ARG BUILD_TOOLS_BASE_IMAGE
+
+FROM ${BUILD_TOOLS_BASE_IMAGE}
+
+# Switch to root user
+USER root
+
+# postgresql (depends on  curl)
+# needed in some tests because neon artifacts cannot be used 
+# in combination with staticaclly built openssl
+# due to https://github.com/neondatabase/neon/issues/8275
+ENV PG_VERSION=16
+RUN set -e \
+    && curl -fsSL 'https://www.postgresql.org/media/keys/ACCC4CF8.asc' | apt-key add - \
+    && echo 'deb http://apt.postgresql.org/pub/repos/apt bullseye-pgdg main' > /etc/apt/sources.list.d/pgdg.list \
+    && apt update \
+    && apt install -y postgresql-client-${PG_VERSION} postgresql-${PG_VERSION} \
```

`postgresql-client-16` is a part of `postgresql-16`

```suggestion
    && apt install -y postgresql-${PG_VERSION} \
```